openquake/server/local_settings.py.template

```diff
@@ -0,0 +1,39 @@
+# This feature requires:
+#  - https://github.com/FirefighterBlu3/python-pam
+#  - https://github.com/cnobile2012/django-pam
+
```

Comment:
Why PAM? I absolutely don't want to have linux users on the server machine except `openquake`, we need to have pure Django authentication not associated to linux users.

Reply:
Ok, I'll create a template specifically for AELO, without PAM.

Reply:
Done